packages/zowe-explorer/src/trees/uss/USSActions.ts

```diff
@@ -60,17 +54,26 @@ export class USSActions {
                 value: node.tooltip as string,
             };
             filePath = await Gui.showInputBox(filePathOptions);
+            node.fullPath = filePath;
         } else {
             filePath = node.fullPath;
         }
+
+        if (filePath == null || filePath.length === 0) {
+            return;
+        }
+
+        const isTopLevel = SharedContext.isSession(node);
```

Comment:
would it make sense to move this above line 42 and reuse it (as with the v2 port)? 😋

Reply:
Yes, sorry I moved this in the v2 port and forgot to apply it here - addressed in 5ded04a